src/plugins/api-proxy/index.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import fs from 'fs';
+import util from 'util';
+import { duotones, getDuotoneUrls } from './util/duotone';
+import getApiProxyRoutes from './routes';
+import proxyApi$ from './proxy';
+
+const API_PROXY_PLUGIN_NAME = 'api-proxy';
+const API_ROUTE_PATH = '/mu_api';
```

Comment:
It's also defined in `src/renderers/server-render.jsx`, so it ends up not really being entirely configurable, right? 

That was more the angle that I was coming from, that we're defining it in two places, but maybe only having it defined once is probably better, and some sort of config seemed like maybe an easy way to do that. 

I think you're right that making it easily configurable probably isn't the way to go, though, since we probably won't want to change it much.